tests/test_lib_events.py

```diff
@@ -909,6 +911,169 @@ def test_02_forward_admin_request(self):
                          "https://remote/token/init")
 
 
+class ResponseManglerTestCase(MyTestCase):
+
+    def test_01_delete_response(self):
+        actions = ResponseManglerHandler().actions
+        self.assertTrue("delete" in actions, actions)
+        self.assertTrue("add" in actions, actions)
+
+        pos = ResponseManglerHandler().allowed_positions
+        self.assertEqual(set(pos), {"post"})
+
+        g = FakeFlaskG()
+        audit_object = FakeAudit()
+        builder = EnvironBuilder(method='POST',
+                                 data={'serial': "SPASS01"},
+                                 headers={})
+        env = builder.get_environ()
+        # Set the remote address so that we can filter for it
+        env["REMOTE_ADDR"] = "10.0.0.1"
+        g.client_ip = env["REMOTE_ADDR"]
+        req = Request(env)
+        req.all_data = {"serial": "SPASS01", "type": "spass"}
+        resp = Response()
+
+        # delete JSON pointer with two components
+        resp.data = """{"result": {"value": true}, "detail": {"message": "Du", "error": 1}}"""
+        options = {"g": g,
+                   "request": req,
+                   "response": resp,
+                   "handler_def": {"options":
+                                       {"JSON pointer": "/detail/message"}
+                                   }
+                   }
+        r_handler = ResponseManglerHandler()
+        res = r_handler.do("delete", options=options)
+        self.assertTrue(res)
+        self.assertEqual(json.loads(resp.data.decode("utf-8")).get("detail").get("error"), 1)
+        self.assertNotIn("message", json.loads(resp.data.decode("utf-8")).get("detail"))
+
+        # delete JSON pointer with one component
+        resp.data = """{"result": {"value": true}, "detail": {"message": "Du", "error": 1}}"""
+        options = {"g": g,
+                   "request": req,
+                   "response": resp,
+                   "handler_def": {"options":
+                                       {"JSON pointer": "/result"}
+                                   }
+                   }
+        r_handler = ResponseManglerHandler()
+        res = r_handler.do("delete", options=options)
+        self.assertTrue(res)
+        self.assertIn("message", json.loads(resp.data.decode("utf-8")).get("detail"))
+        self.assertNotIn("result", json.loads(resp.data.decode("utf-8")))
+
+        # JSON pointer with more than 3 components not supported
```

Comment:
We should also check deleting a JSON pointer with 3 components.

Reply:
Good idea. Please add it.

Reply:
Don in c7225471